src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -198,10 +198,37 @@ private void SetupCommonOptionsForCommands(List<Command> commands)
                 return (int)ExitCode.ErrorInvalidARMTemplate;
             }
 
-            var analysisResult = AnalyzeTemplate(templateFilePath, parametersFilePath);
+            ExitCode exitCode;
+            if (parametersFilePath != null)
+            {
+                exitCode = AnalyzeTemplate(templateFilePath, parametersFilePath);
+            }
+            else
+            {
+                // Check if a parameters.json file is present according to naming standards here https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/parameter-files#file-name, and if so use it as the parametersFile input
```

Comment:
Can we move this comment above FindParameterFileForTemplate instead of having it in two places? Replacing "a parameters.json file is" for "parameters*.json files are"